pkg/runner/runner.go

```diff
@@ -27,7 +27,11 @@ var (
 
 func Run(cfg *Config) error {
 	log.Info("msg", "Version:"+version.Promscale+"; Commit Hash: "+version.CommitHash)
-	log.Info("config", util.MaskPassword(fmt.Sprintf("%+v", cfg)))
+
+	redacted := *cfg
+	redacted.PgmodelCfg.Password = "****"
+	redacted.PgmodelCfg.DbUri = "****"
```

Comment:
The connection string will be logged later if there is an error or not so I'm fine redacting the complete string here. Solution you mentioned doesn't work since its not in that format.